modules/javafx.graphics/src/main/java/javafx/scene/text/Text.java

```diff
@@ -1821,6 +1865,35 @@ final BooleanProperty caretBiasProperty() {
             }
             return caretBias;
         }
+
+        private IntegerProperty tabSize;
+
+        final int getTabSize() {
+            return tabSize == null ? TextLayout.DEFAULT_TAB_SIZE : tabSize.get();
+        }
+
+        final IntegerProperty tabSizeProperty() {
+            if (tabSize == null) {
+                tabSize = new StyleableIntegerProperty(TextLayout.DEFAULT_TAB_SIZE) {
+                    @Override public Object getBean() { return Text.this; }
+                    @Override public String getName() { return "tabSize"; }
+                    @Override public CssMetaData getCssMetaData() {
+                        return StyleableProperties.TAB_SIZE;
+                    }
```

Comment:
The `SimpleStyleableIntegerProperty` subclass can be used instead of `StyleableIntegerProperty` to make the code cleaner:
```
new SimpleStyleableIntegerProperty(StyleableProperties.TAB_SIZE, Text.this, "tabSize", TextLayout.DEFAULT_TAB_SIZE);
```

(This can be done for many places in the library)

Reply:
I'd probably leave it as is for this PR, since it matches other attributes in this file.